tcms/testruns/static/testruns/js/get.js

```diff
@@ -34,8 +34,32 @@ $(document).ready(() => {
         }
     });
 
-    $('.add-hyperlink-bulk').click(function() {
+    $('.add-comment-bulk').click(function() {
         $(this).parents('.dropdown').toggleClass('open');
+
+        const selected = selectedCheckboxes()
+        if ($.isEmptyObject(selected)) {
+            return false
+        }
+
+        const enterCommentText = $('#test_run_pk').data('trans-comment')
+        const comment = prompt(enterCommentText);
+        if (!comment) {
+            return false
+        }
+
+        selected.executionIds.forEach(executionId => {
+            jsonRPC('TestExecution.add_comment', [executionId, comment], () => {
+                delete expandedExecutionIds[expandedExecutionIds.indexOf(executionId)];
```

Comment:
I think that we can still call `animate` here without handler. this way the row will glow and give visual perspective that something has happened with this entry

Reply:
You can do the removal of the ID inside animate if you want, works either way.

Reply:
I get a strange behavior - after expansion of the first row I see 2 editors instead of 1

Reply:
I had that one as well, but it is supposed to be fixed. try now after the rebase?

Reply:
This is still present but isn't unique to this page, the TP page had something similar but in reverse, see:
https://github.com/kiwitcms/Kiwi/pull/2097

The problem in the above PR is how we kept the internal state even after a row has been redrawn b/c that removed the DOM node for the already existing comments editor. 

Here the problem is very likely from a similar root cause but instead of removing the dom node we keep adding more.

Reply:
Also see a1805a1b49134e824a16031c59a9206fff57e706, may be useful

Reply:
the root cause of that problem is that we remove the execution id from the list of rows that are expanded. hence, when we expand it again it call the `getExpandArea` and `initSimpleMDE` again. I solved it by keeping the state of whether the Simple MDE is initialized in a html data attribute. see 9b0690e9f3fdb1c760429bdb3eaf13be5a58eb0a